test/e2e/placement_selecting_resources_test.go

```diff
@@ -138,3 +142,453 @@ var _ = Describe("creating CRP and selecting resources by label", Ordered, func(
 		Eventually(finalizerRemovedActual, eventuallyDuration, eventuallyInterval).Should(Succeed(), "Failed to remove controller finalizers from CRP %s", crpName)
 	})
 })
+
+var _ = Describe("validating CRP when cluster-scoped resources become selected after the updates", Ordered, func() {
```

Comment:
There are a lot of similarities between those cases. Would it make sense to share some code between them?

Reply:
i.e. put the delete to afterAll and share one afterAll.

Reply:
Also, did we place any cluster scoped resource (not namespace)?

Reply:
I tried this structure,

var _ = Describe(..., func (){
         BeforeAll()
         AfterAll()
         var _ = Describe("creating CRP and selecting resources by label", Ordered, func() {
                 It()
                It()
         })
})

However, ginkgo does not allow to put another layer on the top of the "ordered" spec. :( Have already created the cleanupxx related func to share the code.

Reply:
` did we place any cluster scoped resource (not namespace)?`

not yet, could add another test case in the next PR